ui/src/main/webapp/src/app/shared/upload/alternative-upload-queue.component.scss

```diff
@@ -26,6 +26,7 @@ li {
 .progress-bar {
   padding-top: 5px;
   padding-bottom: 5px;
+  color: #363636;
```

Comment:
I agree it is very unreadable when progress bar is empty. It is basically white on grey background with very low contrast. But on the other hand, the black color doesn't solve it very well, because it has worse readability for full progress bars.

IMHO we should design progress bars little bit differently - maybe we could get inspired by patternfly a use some of their patterns.

https://www.patternfly.org/pattern-library/widgets/#progress-bars

I like the "Label on Top Left and Top Right" one with just one change - maybe we could keep percentage inside progress bar.